support/kind/kind.go

```diff
@@ -95,7 +95,7 @@ func (k *Cluster) clusterExists(name string) (string, bool) {
 }
 
 func (k *Cluster) CreateWithConfig(imageName, kindConfigFile string) (string, error) {
-	return k.Create("--image", imageName, "--config", kindConfigFile)
+	return k.Create("--image", fmt.Sprintf("%q", imageName), "--config", kindConfigFile)
```

Comment:
maybe if the image is empty we remove the `--image` flag from the create instead of setting an empty string?

wdyt @vladimirvivien

Reply:
Totally agree, and I would avoid quoting it then, so our hack of using `”\”\””` won’t even break. Just let me know if you want me to switch to:
```golang

func (k *Cluster) CreateWithConfig(imageName, kindConfigFile string) (string, error) {
	args := []string{"--config", kindConfigFile}
	if imageName != "" {
		args = append(args, "--image", imageName)
	}
	return k.Create(args...)
}

```

Reply:
+1 to what @cpanato In case if the image is an empty string, let us just drop the argument. Makes it more readable.

Reply:
I agree, updated the PR accordingly 🙏